docs/software/apps-and-envs/lammps.md

```diff
@@ -18,13 +18,8 @@ There are several LAMMPS modules on Midway2 and Midway3 that you can check via `
 ===+ "Midway3"
     ```
     ---------------------------- /software/modulefiles -----------------------------
```

Comment:
Maybe we can also update the m2 modules' list?

Reply:
Thanks, good point. I need to install the software on m2 first.